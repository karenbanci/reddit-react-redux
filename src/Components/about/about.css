.container-about {
  width: 879px;
  height: 900px;
  background-color: #fff;
  display: flex;
  flex-direction: row;
  margin: 110px 0;
  border-radius: 5px;
}
.main-header-about{
  display: flex;
  justify-content: flex-end;
  width: 100%;
  margin: 10px 15px 0 0 ;
}

.main-header-about button:hover {
  color: #FF4600;
  cursor: pointer;
}

.about-h1 {
  font-size: 2rem;
}

.main h2{
  text-align: justify;
  padding: 10px 40px;
}

.icon-about{
  width: 29px;
  height: 29px;
  margin: 10px;
}

@media only screen and (max-width: 768px) {
  .container-about {
    width: 95%;
    height: 1300px;
    background-color: #fff;
    display: flex;
    flex-direction: row;
    margin: 100px 0;
  }

    .main h1 {
      font-size: 1.3rem;
      padding: 0 20px;
    }

  .main h2 {
    text-align: justify;
    padding: 0 20px;
  }

}
