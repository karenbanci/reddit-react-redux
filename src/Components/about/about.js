import React from "react";
import "./about.css";

import { Link } from "react-router-dom";
import linkedin from "../images/linkedin.png";
import facebookSocial from "../images/facebook-social.png";
import instagram from "../images/instagram.png";

import javascript from "../images/javascript.png";
import react from "../images/react.webp";




import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faXmark} from "@fortawesome/free-solid-svg-icons";


export const About = () => {
return (
  <div className="container-about">
    {/* barra lateral */}
    <div className="side-bar"></div>

    <div className="main">
      <div className="main-header-about">
        <Link to="/">
          <button className="btn-exit-about">
            <FontAwesomeIcon icon={faXmark} />
          </button>
        </Link>
      </div>
      <h1 className="about-h1">About</h1>
      <h2>
        Lorem Ipsum is simply dummy text of the printing and typesetting
        industry. Lorem Ipsum has been the industry's standard dummy text ever
        since the 1500s, when an unknown printer took a galley of type and
        scrambled it to make a type specimen book. It has survived not only five
        centuries, but also the leap into electronic typesetting, remaining
        essentially unchanged. It was popularised in the 1960s with the release
        of Letraset sheets containing Lorem Ipsum passages, and more recently
        with desktop publishing software like Aldus PageMaker including versions
        of Lorem Ipsum.
      </h2>

      <div className="midia-social-contact">
        <Link to="/">
          <img className="icon-about" src={linkedin} alt="midia-linkedin" />
        </Link>
        <Link to="/">
          <img
            className="icon-about"
            src={facebookSocial}
            alt="midia-facebook"
          />
        </Link>
        <Link to="/">
          <img className="icon-about" src={instagram} alt="midia-instagram" />
        </Link>
      </div>

      <h1 className="about-h1">Tecnologies</h1>
      <div className="tecnologies">
        <img className="icon-about" src={javascript} alt="tecnology-javascript" />
        <img
          className="icon-about"
          src={react}
          alt="tecnology-react"
        />
      </div>

      <h3 className="by-karen">By Karen Honorio Banci</h3>
    </div>
  </div>
);
};
