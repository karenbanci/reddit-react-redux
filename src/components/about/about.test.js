import { About } from "./about";
import renderer from "react-test-renderer";

test("testar o Snapshot", () => {
  const aboutFunction = renderer.create(About);
  let result = aboutFunction.toJSON();

  expect(result).toMatchSnapshot();
})
