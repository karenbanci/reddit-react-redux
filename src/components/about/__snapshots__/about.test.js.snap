// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test de  1`] = `null`;

exports[`test de Snapshot 1`] = `null`;

exports[`testar de Snapshot 1`] = `null`;

exports[`testar o Snapshot 1`] = `null`;

exports[`teste de Snapshot 1`] = `null`;
